### ISSUE TITLE
Bad examples for outerHTML and innerHTML

### ISSUE THREAD

**Issue by mavit** [Azq2/perl-html5-dom#6]
The documentation gives the following example:

```perl
my $tree = HTML5::DOM->new->parse('<div id="test">some <b>bold</b> test</div>');
print $tree->outerHTML;                         # <div id="test">some <b>bold</b> test</div>
```

Running this code give the following error:
```
Can't locate object method "outerHTML" via package "HTML5::DOM::Tree"
```

It's tempting to try `$tree->root->outerHTML` instead, but this doesn't give the expected output either since the root node is `<html>`, not `<div>`.

**Comment by mavit** [comment]
A workaround for my use case is `$tree->body->innerHTML`.

**Comment by Azq2** [comment]
@mavit thanks! Fixed in 1.17
